docs/configuration.asciidoc

```diff
@@ -726,6 +726,22 @@ between `0.0` and `1.0`.
 We still record overall time and the result for unsampled transactions, but no
 context information, tags, or spans.
 
+[float]
+[[config-use-experimental-sql-parser]]
+==== `use_experimental_sql_parser`
+|============
+| Environment                               | `Config` key                  | Default
+| `ELASTIC_APM_USE_EXPERIMENTAL_SQL_PARSER` | `use_experimental_sql_parser` | `false`
+|============
+
+Use a newer, more precise but still experimental approach to generating summaries of
+your app's SQL statements.
```

Comment:
Maybe just one sentence _if you don't use this option, your SQL statements will still be summarized albeit less accurately._ or something
Just so users know that their statements will be summarized even if they don't set this option.